std_msgs/Header header
Joints[] joints_array
builtin_interfaces/Time[] t_k #desired execution of the trajectory
